### DISCUSSION TITLE
Blanked out screen/disable backlight option when client is connected

### DISCUSSION THREAD

**Discussion by Halowek** [bk138/droidVNC-NG#228] [Ideas]
hi, I am using this app at my tablet an i was curious if i could control it without someone behind the tablet seeing what i was doing (if enabled) is there a way to build this in?

**Comment by bk138** [comment]
No, that's not possible as there's always some sort of screen-scraping going on for a live session. #131 might be an approach for a headless session, but this will not affect the current on-screen session.

**Comment by StatNoiz** [comment]
I came across a similar need for this but for likely different reasons: at night I wanted to eliminate the glare of my devices that I VNC into...

...my solution was to disable automatic backlight control and put brightness to it's absolute lowest. This doesn't affect your viewer's readability but it will make it difficult to read your device especially in a bright environment without proper backlighting.

That said, having an option to disable your devices backlight entirely...I don't know if that's a possibility for the devs to add.
